File: APIGuanacoBusiness/src/models/job.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;
const jobSchema = new Schema(
  {
    title: {
      type: String,
      required: true,
    },
    logoimag: {
        type: String
    },
    description: {
        type: String,
        required: true  
    },
    schedule: {
        type: String,
        required: true
    },
    salary: {
        type: Number,
        required: true
    },
    worktype: { 
        type: Schema.Types.ObjectId, 
        ref: "worktypes", 
        required: true },

    department:{ 
        type: Schema.Types.ObjectId, 
        ref: "department", 
        required: true },

    workplace: {
        type: String,
        required: true
    },
    employer: { 
        type: Schema.Types.ObjectId, 
        ref: "user" , 
        required: true } ,
    requisites: { 
            type: String, 
            required: true 
        },  
  },
  { timestamps: true }
);

module.exports =  mongoose.model("job", jobSchema);

File: MobileApp/app/src/main/java/com/guanacobusiness/model/job.kt
package com.guanacobusiness.model

import com.google.gson.annotations.SerializedName

data class job(
    @SerializedName("title")val title: String,
    @SerializedName("logoimag")val imagen: String,
    @SerializedName("description")val description: String,
    @SerializedName("schedule")val schedule: String,
    @SerializedName("salary")val salary: Number,
    @SerializedName("worktype")val worktype: worktype,
    @SerializedName("department")val department: department,
    @SerializedName("workplace")val workplace: String,
    @SerializedName("employer")val employer: user,
    @SerializedName("requisites")val requisites: String
)
